### PR TITLE
New experimental cut down pages

### PR DESCRIPTION
**Pull request by johnduffell**
This is just an experiment about cutting down the content to work better on mobile.  To be continued in later PRs. @NataliaLKB

### CODE REVIEW

File: article/app/controllers/ArticleController.scala [hunk 1]
```diff
@@ -34,7 +38,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
   override def renderItem(path: String)(implicit request: RequestHeader): Future[Result] = mapModel(path)(render(_))
 
   private def renderLatestFrom(model: ArticleWithStoryPackage, lastUpdateBlockId: String)(implicit request: RequestHeader) = {
-      val html = withJsoup(BodyCleaner(model.article, model.article.body)) {
+      val html = withJsoup(BodyCleaner(model.article, model.article.body, false)) {
```
Comment (jennysivapalan) [inline article/app/controllers/ArticleController.scala]: I'd recommend adding what the parameter of the false value is for readability e.g 
```
val html = withJsoup(BodyCleaner(model.article, model.article.body, isBlah= false)) {
```

File: article/app/views/articleAMP.scala.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    @defining(model.article) { metaData =>
+    <head>
+        <meta charset="utf-8">
+        <title>@views.support.Title(metaData)</title>
+        <link rel="canonical" href="/@metaData.id" />
+        <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+        <style>
+
+            body {
+                font-family: "Guardian Egyptian Web","Guardian Text Egyptian Web",Georgia,serif;
```
Comment (stephanfowler) [inline article/app/views/articleAMP.scala.html]: Don't need Egyptian...

File: article/app/views/articleAMP.scala.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            .content__dateline {
+                padding: 0.125rem 0 0.375rem;
+                margin: 0;
+            }
+
+            .caption, .content__dateline {
+                font-size: 0.75rem;
+                line-height: 1rem;
+                border-bottom: 0.0625rem dotted #dfdfdf;
+                font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
```
Comment (stephanfowler) [inline article/app/views/articleAMP.scala.html]: Don't need Gu Sans...

File: common/app/implicits/Requests.scala [hunk 4]
```diff
@@ -23,11 +23,9 @@ trait Requests {
 
     lazy val isRss: Boolean = r.path.endsWith("/rss")
 
-    lazy val isWebp: Boolean = {
-      val requestedContentType = r.acceptedTypes.sorted(MediaRange.ordering)
-      val imageMimeType = requestedContentType.find(media => media.accepts("image/jpeg") || media.accepts("image/webp"))
-      imageMimeType.exists(_.mediaSubType == "webp")
-    }
+    lazy val isPcu: Boolean = r.path.endsWith("/amp")
```
Comment (stephanfowler) [inline common/app/implicits/Requests.scala]: `isAmp`?

**Comment by stephanfowler** [comment]
:+1:  nice work!

**Comment by jennysivapalan** [comment]
:+1: Refactorings are a little hard to read in the diff (not your fault) but in general they look much better. Given this affects common and Article I would give this a thorough test in Code.

**Pull request merged**